README.md

```diff
@@ -96,8 +100,70 @@ To run the script, use the following command:
 ```bash
 python converter.py
 ```
-The script will then generate OpenMRS 3 form configurations and translation files from the data in the Excel file, and store them in the folder `generated_form_schemas`. Then you can copy-paste them directly into OpenMRS Initializer folder or Form Builder UI.
 
+The script will generate OpenMRS 3 form configurations and translation files from the data in the Excel file, and store them in the folder `generated_form_schemas`. Then you can copy-paste them directly into OpenMRS Initializer folder or Form Builder UI.
+
+#### Calculation Features
+
+The form generator supports two main types of calculations that can be configured in the Excel metadata file:
+
+1. **Previous Observation Values**: Fetch the most recent observation value for a concept from previous encounters
+2. **Cross-References**: Reference values from other questions within the same form
+
+##### Fetching Previous Values
+
+To fetch a value from previous encounters, add `previous` or `latest` in the Calculation column:
+
+```
+Question         | External ID            | Datatype | Calculation
+-----------------|------------------------|----------|------------
+Last PHQ-9 score | depressionSeverityScale| coded    | previous
```

Comment:
This logic needs to be adjusted to use the calculated column to reference columns